platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegion.java

```diff
@@ -261,6 +262,13 @@ public void onStatusChanged(final OfflineRegionStatus status) {
             public void run() {
               if (observer != null) {
                 observer.onStatusChanged(status);
+
+                if (status.isComplete()) {
+                  TelemetryDefinition telemetry = Mapbox.getTelemetry();
+                  if (telemetry != null) {
+                    telemetry.onOfflineDownloadEndSuccess(definition, status);
+                  }
```

Comment:
above will only  run if there is an observer registered but having an observer registered isn't a requirement, I believe this can be extracted from the `if(observer !=null)` block